lighthouse-core/lib/sd-validation/schema-validator.js

```diff
@@ -96,11 +96,18 @@ function validateObjectKeys(typeOrTypes, keys) {
     // remove Schema.org input/output constraints http://schema.org/docs/actions.html#part-4
     .map(key => key.replace(/-(input|output)$/, ''))
     .filter(key => !allKnownProps.has(key))
-    .map(key => ({
-      message: `Unexpected property "${key}"`,
-      key,
-      validTypes: types,
-    }));
+    .map(key => {
+      return ({
+        message: `Unexpected property "${key}"`,
+        key,
+        validTypes: types.map(typeUri => {
+          const typeNameMatch = typeUri.match(/[\w]+$/);
+          const name = typeNameMatch ? typeNameMatch[0] : typeUri;
+          const uri = typeUri.replace('http://', 'https://');
```

Comment:
what effect does this end up having, just curious :)

Reply:
We don't end up linking to non-https pages from the report.

Reply:
oh, no impact at all from the structured data side of things the https/http are always interchangable?

Reply:
[I think so](https://schema.org/docs/faq.html#19).
> Q: Should we write 'https://schema.org' or 'http://schema.org' in our markup?
> 
> There is a general trend towards using 'https' more widely, and you can already write 'https://schema.org' in your structured data. Over time we will migrate the schema.org site itself towards using https: as the default version of the site and our preferred form in examples. However 'http://schema.org' -based URLs in structured data markup will remain widely understood for the forseeable future and there should be no urgency about migrating existing data. This is a lengthy way of saying that both 'https://schema.org' and 'http://schema.org' are fine.
> 
>